### PR TITLE
Add workflow for testing pre-release files

### PR DESCRIPTION
**Pull request by maxrjones**

### CODE REVIEW

File: .github/workflows/pre-release.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  # schedule:
+  # - cron: '0 0 * * 0'
+
+env:
+  # The GMT version we're testing
+  GMT_VERSION: 6.4.0
+  # disable auto-display of figures
+  GMT_END_SHOW: off
+  # URL of the GMT Bundle
+  GMT_BUNDLE_URL: ftp://ftp.soest.hawaii.edu/release/gmt-6.4.0-darwin-x86_64.dmg
```
Comment (maxrjones) [inline .github/workflows/pre-release.yml]: @PaulWessel, can you comment with the correct path for this?

Comment (seisman) [reply .github/workflows/pre-release.yml]: I think it's `ftp://ftp.soest.hawaii.edu/pwessel/release/gmt-6.4.0-darwin-x86_64.dmg`

Comment (maxrjones) [reply .github/workflows/pre-release.yml]: Thanks

**Comment by maxrjones** [comment]
It looks like the bundle only works for macOS 12, in which case we'll need to update https://github.com/GenericMappingTools/gmt/blob/master/INSTALL.md.

The CI also couldn't find the windows installers on the ftp.

**Comment by PaulWessel** [comment]
Sorry, just placed the Win installers in the pwessel/release directory now.

**Comment by PaulWessel** [comment]
Hm, yes I think when the bundle is built it is built with on the latest OS with the latest frameworks etc.  I dont know how to build for older OS versions - perhaps there are settings for this in cmake.  The Xcode I have comes with the 12.3 SDK.

**Pull request closed without merging**